mod/wikirate_source/spec/set/type/source/source_spec.rb

```diff
@@ -4,10 +4,10 @@
 describe Card::Set::Type::Source do
   describe "while creating a Source" do
     before do
-      login_as 'joe_user' 
+      login_as 'joe_user'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.